File: python/ENEPro/EneParameterRange.py
# encoding: UTF-8

class EneParameterRange :
    #股票代号
    #code=''
    #上限的最高值
    #upperMax=0
    #上限的最低值
    #upperMin=0
    #下限的最高值
    #lowerMax=0
    #下限的最低值
    #lowerMin=0
    #天数的最高值
    #daysMax=0
    #天数的最低值
    #daysMin=0
    def getCode(self) :
        return self.code
    
    def setCode(self,code) :
        self.code = code
    
    def getUpperMax(self) :
        return self.upperMax;
    
    def setUpperMax(self,upperMax) :
        self.upperMax = upperMax
    
    def getUpperMin(self) :
        return self.upperMin
    
    def setUpperMin(self,upperMin) :
        self.upperMin = upperMin
    
    def getLowerMax(self) :
        return self.lowerMax
    
    def setLowerMax(self,lowerMax) :
        self.lowerMax = lowerMax
    
    def getLowerMin(self) :
        return self.lowerMin
    
    def setLowerMin(self,lowerMix) :
        self.lowerMin = lowerMix
    
    def getDaysMax(self) :
        return self.daysMax
    
    def setDaysMax(self,daysMax) :
        self.daysMax = daysMax
    
    def getDaysMin(self) :
        return self.daysMin;
    
    def setDaysMin(self,daysMin) :
        self.daysMin = daysMin
    

File: java/eneProject/src/org/nju/ene/pojo/EneParameterRange.java
package org.nju.ene.pojo;

public class EneParameterRange {
	//股票代号
	private String code;
	//上限的最高值
	private int upperMax;
	//上限的最低值
	private int upperMin;
	//下限的最高值
	private int lowerMax;
	//下限的最低值
	private int lowerMin;
	//天数的最高值
	private int daysMax;
	//天数的最低值
	private int daysMin;
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public int getUpperMax() {
		return upperMax;
	}
	public void setUpperMax(int upperMax) {
		this.upperMax = upperMax;
	}
	public int getUpperMin() {
		return upperMin;
	}
	public void setUpperMin(int upperMin) {
		this.upperMin = upperMin;
	}
	public int getLowerMax() {
		return lowerMax;
	}
	public void setLowerMax(int lowerMax) {
		this.lowerMax = lowerMax;
	}
	public int getLowerMin() {
		return lowerMin;
	}
	public void setLowerMin(int lowerMix) {
		this.lowerMin = lowerMix;
	}
	public int getDaysMax() {
		return daysMax;
	}
	public void setDaysMax(int daysMax) {
		this.daysMax = daysMax;
	}
	public int getDaysMin() {
		return daysMin;
	}
	public void setDaysMin(int daysMin) {
		this.daysMin = daysMin;
	}
	
	
}
